test/functional/FunctionalCodeActionsSpec.hs

```diff
@@ -233,6 +233,31 @@ spec = describe "code actions" $ do
             \foo :: [Int] -> Int\n\
             \foo x = maxBound"
 
+      it "shows more suggestions" $ when ghc84 $
```

Comment:
Can we add parsing for any of the other GHC versions too?

Reply:
Sure, I can look into that